examples/prismic/app/layout.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import { draftMode } from 'next/headers'
+
+import '@/lib/makeswift/components'
+import { MakeswiftProvider } from '@/lib/makeswift/provider'
+
+import './globals.css'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata: Metadata = {
+  title: 'Create Next App',
+  description: 'Generated by create next app',
+}
+
+export default async function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode
+}>) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <MakeswiftProvider previewMode={(await draftMode()).isEnabled}>
```

Comment:
@arvinpoddar, please correct me if I'm wrong here, but am I misremembering that we don't need this anymore after we introduced "native" Draft Mode?